cpp/data/system_updates/103.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -o errexit
+
+
+if [[ $TUEFIND_FLAVOUR != "krimdok" ]]; then
+    exit 0
+fi
+
+if [ -e /var/spool/cron/root ]; then # CentOS
+    root_crontab=/var/spool/cron/root
+else # Ubuntu
+    root_crontab=/var/spool/cron/crontabs/root
+fi
+
+sed --in-place '/^# END VUFIND AUTOGENERATED/i 0 1 1 * * "$BIN/update_print_issns_titles_online_ppns_and_online_issns_list.sh"' \
```

Comment:
1) This is not the same command as in the krimdok.cronjobs installer file (log file missing).
2) It should be inserted at the same position as in krimdok.cronjobs (between expire_searches and purge_old_data), and not before the END VUFIND AUTOGENERATED section